### ISSUE TITLE
editorial: Add table of correspondences for builtin functions in common shader languages

### ISSUE THREAD

**Issue by dneto0** [gpuweb/gpuweb#2502]
I think @mehmetoguzderin suggested that the WGSL spec should have a table showing which WGSL builtin function corresponds to which functions or operations in common languages and shader languages, e.g. HLSL, GLSL.

e.g. ctz in C99 is countTrailingZeros in WGSL

**Comment by mehmetoguzderin** [comment]
True, these can also be a small line next to or under the functions' first introduction. The advantage can be where a table on its own would likely be a mapping, whereas would allow avoiding spelling the names of the referenced common uses, like: "Commonly also known as x, y, z, ..." not naming the API/language directly.

**Comment by kainino0x** [comment]
This can go in the Correspondence Reference https://gpuweb.github.io/gpuweb/correspondence/
